lib/find.js

```diff
@@ -1,13 +1,13 @@
-function find(selector, el) {
-  let element;
-
-  if (el) {
-    element = el;
-  } else {
-    element = document.body;
+function find(selector, context = null) {
+  if (selector instanceof HTMLElement) {
+    return [selector];
+  } else if (selector instanceof NodeList) {
+    return [].slice.call(selector);
+  } else if (typeof selector === 'string') {
+    const startElement = (context) ? find(context) : [document.body];
```

Comment:
I used `find` because I foresee the ability to have the context being multiple elements. For example, being able to find all paragraphs inside any div that has a class of `test-div`.